core/ir/instr_shared.c

```diff
@@ -2988,6 +2989,172 @@ instr_uses_fp_reg(instr_t *instr)
     return false;
 }
 
+instr_t *
+instr_convert_to_isa_regdeps(void *drcontext, instr_t *instr_from)
```

Comment:
This is the conversion function.
Its purpose is to generate and instr_t that decode_from_synth() would have generated.